docs/sql-ref-syntax-aux-cache-uncache-table.md

```diff
@@ -21,11 +21,15 @@ license: |
 
 ### Description
 `UNCACHE TABLE` removes the entries and associated data from the in-memory and/or on-disk cache for a given table or view. The
-underlying entries should already have been brought to cache by previous `CACHE TABLE` operation. `UNCACHE TABLE` on a non-existent table throws Exception if `IF EXISTS` is not specified.
+underlying entries should already have been brought to cache by previous `CACHE TABLE` operation. `UNCACHE TABLE` does not remove 
+the temporary view if it is created by `CACHE TABLE name AS query`. `UNCACHE TABLE` on a non-existent table throws Exception if `IF EXISTS`
```

Comment:
`UNCACHE TABLE` does not remove the temporary view if it is created by `CACHE TABLE name AS query` -> it sounds like it will remove a temporary view that is not created by `CACHE TABLE name AS query`.

Or just say `UNCACHE TABLE` does not remove a temporary view?